api/v1alpha1/oadp_types.go

```diff
@@ -375,6 +375,11 @@ type DataProtectionApplicationSpec struct {
 	// features defines the configuration for the DPA to enable the OADP tech preview features
 	// +optional
 	Features *Features `json:"features"`
+	// Which imagePullPolicy to use in all container images used by OADP.
+	// By default, for images with sha256 or sha512 digest, OADP uses IfNotPresent and uses Always for all other images.
+	// +optional
+	// +kubebuilder:validation:Enum=Always;IfNotPresent;Never
+	ContainerImagePullPolicy *corev1.PullPolicy `json:"containerImagePullPolicy,omitempty"`
```

Comment:
Also, maybe drop the `Container` prefix and just make the spec `ImagePullPolicy` WDYT ?

Reply:
done, thanks :smile_cat: